src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -30,6 +30,77 @@ public async Task TestEmptyFile()
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
         public async Task TestSimpleReferenceType()
+        {
+            await new VerifyCS.Test
+            {
+                LanguageVersion = LanguageVersion.Preview,
```

Comment:
Why is this `Preview` and not the new `CSharpNext`? I understand both are equivalent, but `CSharpNext` should help updating this to CSharp11 when released.

Reply:
It's an oversight. Thanks for catching it.